addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -490,7 +506,25 @@ export async function loadImageInfo(img, attachmentSrc = '') {
     const srcUrl = new URL(src, docHref);
     const relativeSrc = srcUrl.pathname;
 
-    const {original} = await rpc('/web_editor/get_image_info', {src: relativeSrc});
+    const imageOptionsResModel = recordInfo.type === "image" ? recordInfo.resModel : "ir.attachment";
+    const imageOptionsResField = recordInfo.type === "image" ? recordInfo.resField : "datas";
+    let imageOptionsResId = recordInfo.resId;
+    if (recordInfo.type !== "image") {
+        // The image is not a field. Search for a potential id of an attachment
+        // linked to a modified image.
+        const resIdRegex = /\/(?:web\/image|unsplash\/[^/]*?)\/(\d+|[^\s/]+)/;
+        const matchedSrc = src.match(resIdRegex);
+        imageOptionsResId = matchedSrc && matchedSrc[1];
+    }
+    // Transform the res_id into a number if possible.
+    imageOptionsResId = isNaN(imageOptionsResId) ? imageOptionsResId : parseInt(imageOptionsResId);
+
```

Comment:
Just a suggestion: you did something similar in python for the `/web_editor/modify_image` route: can't you define a function in python in charge of this "conversion" (`if restype===...`)  ? It would then be used by both routes in that case :slightly_smiling_face: 
It would allow `loadImageInfo` to just send the information (making it simpler) and the route will take care of processing this info :+1: (you will need to pass the type in the request though)